### DISCUSSION TITLE
Version 3.0.0-beta release in master branch

### DISCUSSION THREAD

**Discussion by mrcodetastic** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#328] [General]
Let me know of the issues you face here. Main changes

* Refactored the entire code base & removed some legacy junk..
* Implemented ESP-S3 support

Known issues:
* Double buffering may not be perfect.

**Comment by EpicLPer** [comment]
Heya! First time LED Matrix user here, and I'm like 99% sure it's not the fault of 3.0.0 :)  
Tried this yesterday on a AZ-Delivery ESP32-WROOM-32 and it seems like it didn't work, but again I think I'm doing something wrong myself here. Never used PlatformIO much before either.

I've changed the board to `az-delivery-devkit-v4` and connected the HUB75 connections to the G-Pins on the ESP according to the pins given in the commented section, including the `PIN_E` a bit below but the display didn't want to work at all.

**Comment by mrcodetastic** [comment]
Can you try the 2.0.7 release and see if you have issues? Should be a drop in replacement.

If you do, can you provide the compile time output when using 3.0.0. Thank you.

**Reply by EpicLPer** [reply]
I've tried with the current release still, and it seems something else went wrong while using the connections on the Matrix. The code works just fine! Think I was simply too stupid to use PIO :)

**Reply by mrcodetastic** [reply]
So you got it working? 🙂

**Reply by EpicLPer** [reply]
Yep, worked perfectly with the included Arduino IDE examples!

**Comment by EpicLPer** [comment]
Do you happen to know if some "cheaper" LED Matrixes have some misprint on their silkscreen? Cause mine says "GND" instead of "D", but it isn't a Ground connection. It's supposed to be "D".

Can try the older version later when I'm home again! :)

**Comment by DarrylStrong** [comment]
Test with a mulitmeter?
Also, what scan is your module? 8,16,32:1?

**Reply by EpicLPer** [reply]
Datasheet says 1/32, which I did use the extra pin for in the config.

**Comment by DarrylStrong** [comment]
Best to check with a multimeter then. Or oscilloscope if you have one :)
